plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -178,6 +179,7 @@ fu_thunderbolt_device_to_string (FuDevice *device, guint idt, GString *str)
 	fu_common_string_append_kb (str, idt, "Safe Mode", self->safe_mode);
 	fu_common_string_append_kb (str, idt, "Native mode", self->is_native);
 	fu_common_string_append_ku (str, idt, "Generation", self->gen);
+	fu_common_string_append_kv (str, idt, "AuthAttribute", self->authenticate_attribute);
```

Comment:
I stumbled over this when I first read it but I cannot think how to better name that is now horrendously long. I was thinking maybe `AuthMethod`/`AuthMechanism`? Maybe just good as is.

Reply:
thanks fixed